dao.py

```diff
@@ -0,0 +1,49 @@
+from flask_pymongo import PyMongo
+from bson.objectid import ObjectId
+
+from app import app
+
+DB_NAME = 'schedule_practice'
+DB_USERNAME = 'admin'
+DB_PASSWORD = 'admin'
+DB_URI = 'mongodb://%s:%s@ds239387.mlab.com:39387/schedule_practice' % (DB_USERNAME, DB_PASSWORD)
```

Comment:
What is this connecting to ?

Reply:
A sandbox DB on a tool called mLab which just hosts DBs for free.

Reply:
Got it, lets begin to initialize the mongodb container in this repo with docker-compose